src/internal/wasi/clocks/v0.2.0/monotonic-clock/monotonic-clock.wit.go

```diff
@@ -78,7 +78,7 @@ func wasmimport_Resolution() (result0 uint64)
 // SubscribeInstant represents the imported function "subscribe-instant".
 //
 // Create a `pollable` which will resolve once the specified instant
-// occured.
+// occurred.
```

Comment:
Sadly this is a misspelling upstream, *and* this is a generated file.  We should probably ignore `src/internal/wasi` for now.

Reply:
ignored